CONTRIBUTING.md

```diff
@@ -11,18 +11,7 @@ Your contributions will fall into two categories:
     - Search for your issue here: https://github.com/pytorch/serve/issues (look for the "good first issue" tag if you're a first time contributor)
     - Pick an issue and comment on the task that you want to work on this feature.
     - To ensure your changes doesn't break any of the existing features run the sanity suite as follows from serve directory:
-        - Install dependencies (if not already installed)
-          For CPU
-
-          ```bash
-          python ts_scripts/install_dependencies.py --environment=dev
```

Comment:
Upstream may pick up on this change, but it makes sense to me if this is duplicate.

Reply:
Yeah, it seems redundant to have the same thing twice in the same file. Let's see if they agree upstream